packages/mobile/src/components/audio-rewards/audioTier.ts

```diff
@@ -27,31 +28,29 @@ export const badgeTiers: { tier: BadgeTier; minAudio: BN }[] = [
   }
 ]
 
-export const tierLevels: BadgeTier[] = [
+export const audioTierOrder: AudioTier[] = [
   'none',
   'bronze',
   'silver',
   'gold',
   'platinum'
 ]
 
-export const getTierLevel = (tier: BadgeTier) => {
-  return tierLevels.indexOf(tier)
+export const getAudioTierRank = (tier: AudioTier) => {
+  return audioTierOrder.indexOf(tier)
 }
 
-export const getBadgeTier = (
+export const getUserAudioTier = (
```

Comment:
might be worth adding a test for this util - up to you though depending on whether you think it's prone to regressions

Reply:
oh tests... would love to add them in mobile at some point, we don't have any im pretty sure 🗡️

Reply:
ohhh haha